src/common/modals/InstanceManager/Overview.js

```diff
@@ -86,6 +102,11 @@ const Overview = ({ instanceName }) => {
     config?.javaArgs
   );
   const [newName, setNewName] = useState(instanceName);
+  const [height, setHeight] = useState(config?.resolution?.height);
+  const [width, setWidth] = useState(config?.resolution?.width);
+  const [gameResolutionSwitch, setGameResolutionSwitch] = useState(
+    config?.resolution?.height || config?.resolution?.width
+  );
```

Comment:
Actually we want to make sure that both are set, since one can't exist without the other